jumpscale/packages/threebot_deployer/package.toml

```diff
@@ -0,0 +1,36 @@
+name = "threebot_deployer"
+ports = [ 80, 443]
+is_auth = true
+is_admin = false
+frontend = "/threebot_deployer"
+
+[[static_dirs]]
+name = "frontend"
+path_url = ""
+path_location = "frontend/"
+index = "index.html"
+spa = true
+
+[[bottle_servers]]
+name = "main"
+file_path = "bottle/solutions.py"
+path_url = "/api/"
+path_dest = "/api/"
+host = "0.0.0.0"
+port = 5560
+
+
+[[servers]]
+name = "threebot_deployer_root_proxy"
+domain = "deploy3bot2.grid.tf"
```

Comment:
deploy3bot.grid.tf is the official domain

Reply:
changed it to `deploy3bot.grid.tf`